TeamCode/src/main/java/org/firstinspires/ftc/teamcode/freightfrenzy2021/opmodes/AutonStates/StateCollectFreightWithEncoders.java

```diff
@@ -64,6 +64,9 @@ public StateCollectFreightWithEncoders(EbotsAutonOpMode autonOpMode){
         freightDetector = new FreightDetector();
```

Comment:
This needs to receive the FreightDetector that was instantiated in the autonOpMode.  Also, we need to add Null protection to any calls to to the freightDetector in case there is an issue with the OpenCV thread.